src/app/mail/MailRecipients.ts

```diff
@@ -44,61 +44,92 @@ export function getRecipients(
 	includeEditors : boolean = false
 ) : Promise <Array <string>>
 {
+
+	// override emails for development
 	if (override && !app.isProduction) {
-		return Promise.resolve([ override ]);
+		return Promise.resolve([override]);
 	}
 
-	// If feedback email override to dedicated addresses is set for website
-	// then send messages there only instead of article authors
-	if (website.feedbackEmailOverride && website.feedbackEmailOverride.length) {
-		return Promise.resolve(website.feedbackEmailOverride);
+	const {
+		name,
+		chiefEditors,
+		overrideSettings: {
+			settings = {
+				feedback : false,
+			},
+			overrides = {
+				feedbackEmail: [],
+				fallbackFeedbackEmail: [],
+			},
+		} = {},
+	} = website;
+
+	const {feedbackEmail = [], fallbackFeedbackEmail = []} = overrides;
+
+	// if website is set to send all feedback's to dedicated addresses then override email addresses
+	if (settings.feedback && feedbackEmail.length) {
+		return Promise.resolve(overrides.feedbackEmail);
```

Comment:
Why do you refer to the property through overrides? You can simply write feedbackEmail